### ISSUE TITLE
Performance of training JPEG-Ti from scratch on ImageNet-1K

### ISSUE THREAD

**Issue by yo3nglau** [JeongsooP/RGB-no-more#1]
Hi ~
Thank you for your salient work!

After reproducing the training from scratch on ImageNet-1K, I got a **slightly lower Val Acc (74.4%)** against the paper (75.1%). I would like to know **if 0.7% lies in the rational range**.

Reproduction details:

I gained ImageNet-1K at https://www.kaggle.com/competitions/imagenet-object-localization-challenge/data.
(I argue that the data are identical ones that at https://image-net.org/challenges/LSVRC/2012/2012-downloads.php, if catastrophe kindly tell me.) Corresponding preprocesses (extract, resize) were finished with the suggested commands and scripts.

_Devices:_
CPU: 96 AMD EPYC 7402 24-Core Processor
GPU: 6 NVIDIA GeForce RTX 3090

_Commands:_
Keep suggested commands unchanged except `--num_cpus=96 --num_gpus=6`

_Performances:_
Training from scratch on ImageNet-1K
[Test Acc: 74.44 %, Test loss: 1.0161 ]
[Val Acc: 78.00 %, Val loss: 0.8591, Train Acc (eval mode): 88.20 %, Train loss (eval mode): 0.4389 ]

Pretrained JPEG-Ti model
[Test Acc: 75.11 %, Test loss: 0.9988 ]
[Val Acc: 78.03 %, Val loss: 0.8516, Train Acc (eval mode): 88.88 %, Train loss (eval mode): 0.4168 ]

Any response will be appreciated.

Best regards

**Comment by JeongsooP** [comment]
Hello, thank you for the detailed report!

I think it is normal; some variance in accuracy is expected. I also noticed some differences in accuracy across different hardware settings (e.g. Titan Xp ↔ 2080 Ti ↔ A40, up to 1%) during experiments.

I trained JPEG-Ti models using 8x 2080Ti and 32x Intel CPU threads. Unless you're using my exact hardware, there could be some differences in randomness/data augmentation/etc even with the same seed.

You can try to change the seed (`--seed`) and see if it helps. But I think the result you're seeing is normal.

**Comment by yo3nglau** [comment]
Thanks for quick reply! I got what you mean.
